pkg/skaffold/debug/transform_python_test.go

```diff
@@ -146,13 +146,14 @@ func TestPythonTransformer_IsApplicable(t *testing.T) {
 
 func TestPythonTransformer_Apply(t *testing.T) {
 	tests := []struct {
-		description   string
-		containerSpec v1.Container
-		configuration imageConfiguration
-		shouldErr     bool
-		result        v1.Container
-		debugConfig   ContainerDebugConfiguration
-		image         string
+		description       string
+		containerSpec     v1.Container
+		configuration     imageConfiguration
+		shouldErr         bool
+		result            v1.Container
+		debugConfig       ContainerDebugConfiguration
+		image             string
+		overrideProtocols []string
```

Comment:
Move this above `shouldErr` since it's an input to the test — and then move the uses below too?